model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -337,24 +337,19 @@ message Annotation {
 // MonitoringInfo protos.
 message MonitoringInfoSpecs {
   enum Enum {
-    // TODO(ajamato): Add the PTRANSFORM name as a required label after
-    // upgrading the python SDK.
-    USER_COUNTER = 0 [(monitoring_info_spec) = {
-      urn: "beam:metric:user:",
-      type_urn: "beam:metrics:sum_int_64",
-    }];
-
-    ELEMENT_COUNT = 1 [(monitoring_info_spec) = {
+    ELEMENT_COUNT = 0 [(monitoring_info_spec) = {
       urn: "beam:metric:element_count:v1",
       type_urn: "beam:metrics:sum_int_64",
+      // TODO(ryan): we currently also generate this metric with ["PTRANSFORM"] labels, but it fails validation in
```

Comment:
That's in python @Ardagan is fixing that implementation to emit the PCollection name. This is the one spec runners should support.

Reply:
OK. I'll leave this for now, perhaps I can remove it before this gets merged, if @Ardagan fixes it first, or lmk what you suggest!